### PR TITLE
Add Envoy security patch details for OSM and Istio addons

### PR DESCRIPTION
**Pull request by keithmattix**

### CODE REVIEW

**Review by justindavies — APPROVED** [review]

File: examples/envoy-ghsa-jfxv-29pc-x22r/get-osm-envoy-version.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+kubectl get deploy -nkube-system osm-injector -ojson | jq '.spec.template.spec.containers[].env[] | select(.name == "OSM_DEFAULT_ENVOY_IMAGE")'
```
Comment (shashankbarsin) [inline examples/envoy-ghsa-jfxv-29pc-x22r/get-osm-envoy-version.sh:3]:
```suggestion
kubectl get deploy -n kube-system osm-injector -o json | jq '.spec.template.spec.containers[].env[] | select(.name == "OSM_DEFAULT_ENVOY_IMAGE")'
```

**Review by shashankbarsin — APPROVED** [review]

File: examples/envoy-ghsa-jfxv-29pc-x22r/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Am I required to take any action?
+Once your cluster has the patch, you must restart all sidecar injected applications in order to replace the vulnerable Envoy sidecars with patched ones. This applies for both the OSM and Istio addons. If your AKS version is [out of support](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions), you will NOT receive a new version of either addon containing the security patch. We strongly recommend you upgrade in order to receive support and security patches.
```
Comment (shashankbarsin) [inline examples/envoy-ghsa-jfxv-29pc-x22r/README.md:10]:
```suggestion
Once your cluster has the patch, you must restart all sidecar injected applications in order to replace the vulnerable Envoy sidecars with patched ones. This applies for both the OSM and Istio addons. If your AKS version is [out of support](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions), you will NOT receive a new version of either addon containing the security patch. We strongly recommend you upgrade in order to receive support and security patches.
```

File: examples/envoy-ghsa-jfxv-29pc-x22r/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Who is at risk?
+All AKS clusters with the OSM or Istio addosn enabled and sidecars injected in their applications
+
+Mitigation Steps?
+The AKS team is rolling out patched Envoy versions to all supported AKS clusters (i.e. versions 1.24 and higher) using the OSM or Istio addons. These updates are expected to be rolled out to all regions in 10 days (July 30th). If you're using the OSM addon, you can use the `get-osm-envoy-version.sh` script in this directory to check your injected Envoy version (requires `jq`). If the image tag is v1.26.3 or v1.25.8, you have the patch and you are not at risk. If you are using the Istio addon, you can use the `get-istio-versio.sh` script in this directory to check your Istio version (also requires `jq`). If the image tag is v1.17.4, then you have the patch and are not at risk.
```
Comment (ritazh) [inline examples/envoy-ghsa-jfxv-29pc-x22r/README.md:7]: get-istio-version.sh

Comment (keithmattix) [reply examples/envoy-ghsa-jfxv-29pc-x22r/README.md:7]: Ah, good catch.

**Pull request merged**